apps/developer-portal/src/components/identity-providers/utils/attribute-settings-utils.tsx

```diff
@@ -152,21 +152,34 @@ export const initSubjectAndRoleURIs = (initialClaims, setSubjectClaimUri, setRol
 
 export const handleAttributeSettingsFormSubmit = (idpId: string, values: IdentityProviderClaimsInterface,
                                                   roleMapping: IdentityProviderRoleMappingInterface[],
-                                                  onUpdate: (idpId: string) => void, dispatch: Dispatch<any>): void => {
+                                                  onUpdate: (idpId: string) => void): void => {
     updateClaimsConfigs(idpId, values)
         .then(() => {
-            dispatch(addAlert({
-                description: "Successfully updated attribute configurations.",
+            store.dispatch(addAlert({
+                description: I18n.instance.t("devPortal:components.idp.notifications." +
+                    "updateClaimsConfigs.success.description"),
                 level: AlertLevels.SUCCESS,
-                message: "Update successful"
+                message: I18n.instance.t("devPortal:components.idp.notifications.updateClaimsConfigs.success.message")
             }));
             onUpdate(idpId);
         })
-        .catch(() => {
-            dispatch(addAlert({
-                description: "An error occurred while the updating claims configurations.",
+        .catch((error) => {
+            if (error.response && error.response.data && error.response.data.description) {
+                store.dispatch(addAlert({
+                    description: I18n.instance.t("devPortal:components.idp.notifications." +
+                        "updateClaimsConfigs.error.description",
+                        { description: error.response.data.description }),
+                    level: AlertLevels.ERROR,
+                    message: I18n.instance.t("devPortal:components.idp.notifications.updateClaimsConfigs.error.message")
```

Comment:
Possible line length issue here. Please check.

Reply:
Not according to Webstorm.